### ISSUE TITLE
Syncroscope "raise", "Lower" buttons not working

### ISSUE THREAD

**Issue by Rastur186** [loiste-interactive/infra-issues#1165]
Trying to sync the 3 generators and the buttons have no effect. Tried an earlier save but still does not work.

**Comment by CorruptComputer** [comment]
They do, you just have to hit them a LOT. I think it's a little overboard how little affect the buttons have, and this leads to the area taking way longer then necessary.

**Comment by Rastur186** [comment]
I hit them continuously for 5 minutes to no effect.

**Comment by Yuccaa** [comment]
What exactly did you do? Did you enter the area normally (no noclipping etc.), turned the genrators on and then tried the synchroscopes?

**Comment by Rastur186** [comment]
Exactly, no clipping, no cheating of any kind. Just entered the area and turned the generators on and went to the control room with the buttons. This is such a great game and I’ve come so far, I hate to end it here.

**Comment by TeamSpen210** [comment]
What's appearing on the 'speed' dial when you do it? Perhaps you're hitting the min/max speed.

**Comment by Rastur186** [comment]
That’s not the case and I’ve given up on this and am ready to move on to something else. Thank you all for your time.

**Comment by Yuccaa** [comment]
I remember someone else encountering this exact bug before.  This seems to be an extremely rare issue.

Try loading an earlier save and see if it fixes the issue. If not, try restarting the chapter.

For other DEVs: I guess this has to do with the vsicpt associated with the synchroscopes.

**Comment by Zuntti** [comment]
Hello,
Could you send me the save files? You can find your saved games by right-clicking INFRA on Steam and selecting "Properties", "Local Files" and "Browse Local Files...". The saves are stored in the "infra/SAVE" folder. I only need the ".sav" files. Can you send those files to contact@loisteinteractive.com
